src/components/TooltipIcon/TooltipIcon.spec.tsx

```diff
@@ -0,0 +1,34 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from "react";
+import { mount } from "@cypress/react";
+import { TooltipIcon } from "./TooltipIcon";
+import { IconSize } from "@foundation/Icon/IconSize";
+
+const TOOLTIP_PROPS = {
+    content: "Lorem ipsum dolor sit amet.",
+};
+const TOOLTIP_ICON_ID = "[data-test-id=tooltip-icon]";
+const TOOLTIP_ICON_TRIGGER_ID = "[data-test-id=tooltip-icon-trigger]";
+const TOOLTIP_ID = "[data-test-id=tooltip]";
+
+describe("TooltipIcon Component", () => {
+    it("should render an icon with default size", () => {
+        mount(<TooltipIcon tooltip={TOOLTIP_PROPS} />);
+
+        cy.get(TOOLTIP_ICON_ID).find(TOOLTIP_ICON_TRIGGER_ID).children().should("have.class", "tw-h-4");
+    });
+
+    it("should render an icon with custom size", () => {
+        mount(<TooltipIcon tooltip={TOOLTIP_PROPS} iconSize={IconSize.Size20} />);
+
+        cy.get(TOOLTIP_ICON_ID).find(TOOLTIP_ICON_TRIGGER_ID).children().should("have.class", "tw-h-5");
+    });
+
+    it("should render a tooltip when the icon is hovered", () => {
+        mount(<TooltipIcon tooltip={TOOLTIP_PROPS} />);
+
+        cy.get(TOOLTIP_ICON_ID).realHover({ position: "top" });
```

Comment:
```suggestion
        cy.get(TOOLTIP_ICON_TRIGGER_ID).realHover({ position: "top" });
```

Reply:
Thanks for the review, @jonasbuechel! 💪 

Using `TOOLTIP_ICON_TRIGGER_ID`, can you get the test to pass every time? I tried but got flaky results, and I'm trying now to understand what causes it 🤷

Reply:
Looks like focus can randomly get lost in some cases. To ensure it won't, I added a click on the entire "body" before the hover action is performed. This seems to make it stable.

Reply:
I think it also depends if you run the tests headless (`npm run test`) or with an open browser (`npm run test:browser`) 🤔